e2e/utils.go

```diff
@@ -137,8 +137,8 @@ func getMons(ns string, c kubernetes.Interface) []string {
 	svcList, err := c.CoreV1().Services(ns).List(opt)
 	Expect(err).Should(BeNil())
 	services := make([]string, 0)
-	for _, svc := range svcList.Items {
-		s := fmt.Sprintf("%s.%s.svc.cluster.local:%d", svc.Name, svc.Namespace, svc.Spec.Ports[0].Port)
+	for i := range svcList.Items {
```

Comment:
@Madhu-1 svc is more readable and meaningful, Isnt it ?

Reply:
same here, this was the issue found in static check